data-upgrade-ecms/src/main/java/org/exoplatform/ecms/upgrade/content/NodeContentUpgradePlugin.java

```diff
@@ -0,0 +1,88 @@
+package org.exoplatform.ecms.upgrade.content;
+
+import org.apache.commons.lang3.StringUtils;
+import org.exoplatform.commons.api.settings.SettingService;
+import org.exoplatform.commons.upgrade.UpgradeProductPlugin;
+import org.exoplatform.container.xml.InitParams;
+import org.exoplatform.services.cms.drives.ManageDriveService;
+import org.exoplatform.services.jcr.RepositoryService;
+import org.exoplatform.services.jcr.ext.common.SessionProvider;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+
+import javax.jcr.Node;
+import javax.jcr.NodeIterator;
+import javax.jcr.Session;
+import javax.jcr.Workspace;
+
+public class NodeContentUpgradePlugin extends UpgradeProductPlugin {
+
+    private static final Log LOG = ExoLogger.getLogger(NodeContentUpgradePlugin.class);
+    private static final String OLD_NODE_PATH = "old.nodePath";
+    private static final String NEW_NODE_PATH = "new.nodePath";
+    private static final String WORKSPACE_COLLABORATION = "collaboration";
+    private String oldNodePath;
+    private String newNodePath;
+    private final RepositoryService repositoryService;
+    private final ManageDriveService manageDriveService;
+
+    public NodeContentUpgradePlugin(SettingService settingService, 
+                                    InitParams initParams, 
+                                    RepositoryService repositoryService, 
+                                    ManageDriveService manageDriveService) {
+        super(settingService, initParams);
+        this.repositoryService = repositoryService;
+        this.manageDriveService = manageDriveService;
+        if (initParams.containsKey(OLD_NODE_PATH)) {
+            oldNodePath = initParams.getValueParam(OLD_NODE_PATH).getValue();
+        }
+        if (initParams.containsKey(NEW_NODE_PATH)) {
+            newNodePath = initParams.getValueParam(NEW_NODE_PATH).getValue();
+        }
+    }
+
+    @Override
+    public void processUpgrade(String s, String s1) {
+        if (StringUtils.isBlank(oldNodePath)) {
+            LOG.error("Couldn't process upgrade, the parameter '{}' is mandatory", OLD_NODE_PATH);
+            return;
+        }
+        if (StringUtils.isBlank(newNodePath)) {
+            LOG.error("Couldn't process upgrade, the parameter '{}' is mandatory", NEW_NODE_PATH);
+            return;
+        }
+        SessionProvider sessionProvider = null;
+        LOG.info("Start upgrade of node");
+        try {
+            sessionProvider = SessionProvider.createSystemProvider();
+            Session session = sessionProvider.getSession(WORKSPACE_COLLABORATION,
+                    repositoryService.getCurrentRepository());
+            Workspace workspace = session.getWorkspace();
+            Node oldNode = (Node) session.getItem(oldNodePath);
```

Comment:
why dont you do 
workspace.move(oldPath, newPath) ? 
Then it should rename the folder Security to Quarantine, and all subnode nodes will move automatically

Reply:
By doing this , the quarantine node will be added twice because i updated the configuration of node [here](https://github.com/exoplatform/ecms/pull/1094/files#diff-d5aac7fbda43eef0e7e613ce8002f673c9b57e093c85323c5da5d66e8eda45d0R34)

Reply:
I think we could do that : 
- if node quarantine exists, drop it
- move security to quarantine